<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>560</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_run">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>450</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>450</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_loading">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>496</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Loading feedback</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>322</x>
      <y>184</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>298</x>
      <y>184</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>273</x>
      <y>184</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>184</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>223</x>
      <y>184</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>199</x>
      <y>184</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>184</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>184</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>184</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_importData2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>57</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Please select a folder.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_importData">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>37</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>No data imported.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_resInterval">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>158</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Resolution Interval</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>228</y>
      <width>105</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Generate STL file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_importData">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>37</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Import Data</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>101</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="suffix">
     <string> mm</string>
    </property>
    <property name="minimum">
     <double>0.010000000000000</double>
    </property>
    <property name="maximum">
     <double>999999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>158</y>
      <width>231</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_yInterval">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>108</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y Interval</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>347</x>
      <y>184</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_leveling">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>198</y>
      <width>105</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Leveling</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_custom_axis">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>105</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Custom Axis:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_size">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="suffix">
     <string> x</string>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>5.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_size">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>407</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Size:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_x_min">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="suffix">
     <string> min</string>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_x_max">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="suffix">
     <string> max</string>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_x_axis">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>X axis:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_y_axis">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Y axis:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_y_max">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="suffix">
     <string> max</string>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_y_min">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="suffix">
     <string> min</string>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_z_axis">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Z axis:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_z_max">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="suffix">
     <string> max</string>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_z_min">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="suffix">
     <string> min</string>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>999999.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
